src/utils/SetupUtil.js

```diff
@@ -82,6 +82,9 @@ export default {
           machineVersion
         });
 
+        virtualBox.getShareDir(machine.name());
```

Comment:
haha no such thing as the master, just an observation. I think you're right, the user would add more folders and restart or start kitematic after making the changes. 
When in doubt restart, right?